docs/en/interfaces/third-party_gui.md

```diff
@@ -14,6 +13,23 @@ Main features:
 
 [Tabix documentation](https://tabix.io/doc/).
 
+
+____
+
+
+
+<img src="https://svgshare.com/i/6W0.svg" alt="" data-canonical-src="https://svgshare.com/i/6W0.svg" width="100" />
+
 ## HouseOps
 
 [HouseOps](https://github.com/HouseOps/HouseOps) is a unique Desktop ClickHouse Ops UI / IDE for OSX, Linux and Windows.
+
+### Features:
+- Query builder;
+- Database manangement (soon);
+- Users manangement (soon);
+- Real-Time Data Analytics (soon);
+- Cluster/Infra monitoring (soon);
+- Cluster manangement (soon);
+- Kafka and Replicated tables monitoring (soon);
+- And a lot of others features (soon) for you take a beautiful implementation of ClickHouse S2.
```

Comment:
What is `ClickHouse S2`? Is it just a https://www.urbandictionary.com/define.php?term=s2
Probably not the best place.